codegen/projections/white_label/spec/endpoints_spec.rb

```diff
@@ -18,14 +18,6 @@ module WhiteLabel
     describe '#endpoint_with_host_label_operation' do
       let(:label) { 'input_label' }
 
-      it 'raises when missing host label member' do
```

Comment:
This test needs to come back. I think I deleted this when I deleted the raising functionality, but Alex and I talked about keeping it in case someone uses `validate_input: false` (we still want to validate URI members). Would you be able to revive it in this PR?